### ISSUE TITLE
appveyor vcpkg seemingly corrupts itself

### ISSUE THREAD

**Issue by MarcoFalke** [bitcoin/bitcoin#19440]
This is the second time this week, see https://ci.appveyor.com/project/DrahtBot/bitcoin/history

Failure looks like:

```
msbuild /p:TrackFileAccess=false build_msvc\bitcoin.sln /m /v:q /nologo
C:\projects\bitcoin\src\util\url.cpp(7,10): fatal error C1083: Cannot open include file: 'event2/http.h': No such file or directory [C:\projects\bitcoin\build_msvc\libbitcoin_util\libbitcoin_util.vcxproj]
C:\projects\bitcoin\src\script\standard.h(12,10): fatal error C1083: Cannot open include file: 'boost/variant.hpp': No such file or directory [C:\projects\bitcoin\build_msvc\libbitcoin_common\libbitcoin_common.vcxproj]

**Comment by sipsorcery** [comment]
I'm starting to think the 15 minutes saved by having the cache isn't justified if the cache is regularly breaking the whole build.

imho #19431 is a good idea after all.

**Comment by sipsorcery** [comment]
I had a look at the appveyor logs for the builds before and after the failures started and there doesn't appear to be anything in the jobs that affected the `vcpkg` dependencies. Seems that between jobs `master.22410` and `master.22411` the cached `C:\tools\vcpkg\installed` directory got corrupted.

We could do something like add a checksum for each cached directory but that ends up being one more step to go wrong. I think it's worth commenting out the cache parts of the appveyor job and seeing how it goes for a while.

**Comment by MarcoFalke** [comment]
I am not sure if #19431 helps with regard to this issue. If the cache corrupts while being stored on the appveyor servers, then there is nothing we can do. If the cache corrupts while running our appveyor script, then #19431 might help. (or disabling the cache altogether will help as well)

**Comment by MarcoFalke** [comment]
Fixed by #19444
